test/deploy/windows/kafka/install/roles/prepare/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- block:
+    - name: Starting Zookeper
+      win_shell: |
+        c:\kafka\bin\windows\zookeeper-server-start.bat c:\kafka\config\zookeeper.properties
+      async: 5
```

Comment:
Same feedback, why timing out at 5s, instead of letting it run until completion?

Reply:
@Julien4218  Same as above, the script never "finishes", since starting a server in the shell, thus the ansible process doesn't finish.  So, I just put 5 seconds in, so it can move to the next process.

Reply:
For windows, see some fire and forget example with powershell, here are a few examples https://stackoverflow.com/questions/49017321/powershell-build-step-fire-and-forget